mlflow/protos/service.proto

```diff
@@ -585,6 +585,20 @@ service MlflowService {
     };
   }
 
+  // Get Trace (returning a TraceInfoV3). Differences between GetTraceInfoV4 and GetTraceInfoV3
+  // are location in the request path, which describes where the Trace is stored.
+  rpc getTraceInfoV4 (GetTraceInfoV4) returns (GetTraceInfoV4.Response) {
```

Comment:
@serena-ruan  Can we change this to instead return a TraceInfoV3 directly to follow AIP guidelines:

```
  rpc getTraceInfoV4 (GetTraceInfoV4) returns (TraceInfoV4) {
```